notebooks/official/automl/sdk_automl_video_action_recognition_batch.ipynb

```diff
@@ -29,9 +29,10 @@
         "id": "title"
```

Comment:
<p>Please review instructions. This all needs to be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>